test/pylib/scylla_cluster.py

```diff
@@ -190,7 +190,7 @@ class ScyllaServer:
     """Starts and handles a single Scylla server, managing logs, checking if responsive,
        and cleanup when finished."""
     # pylint: disable=too-many-instance-attributes
-    START_TIMEOUT = 300     # seconds
+    START_TIMEOUT = 1000     # seconds
```

Comment:
I still think 1000 seconds is overkill.
I suggest we don't increase the timeout more than twice from what it was. It was 5 minutes, so let's increase it to 10 minutes, i.e. 600 seconds.

Reply:
I intentionally chose a bigger number: 
1)  we can pass the CI test before we fix the CI slowness now
2)  we can choose a more reasonable number later